### ISSUE TITLE
error when add new relay: ex infura

### ISSUE THREAD

**Issue by dirmansebastian** [LoopringSecondary/loopr#400]
hey,

is there any way to change the relay to my local node or infura? 

![image](https://user-images.githubusercontent.com/15945292/41626758-fc37cece-7425-11e8-8e2f-3b67a6bf3989.png)

getting error wss://mainnet.infura.io/socket.io/?EIO=3&transport=websocket when i add relay url: https://mainnet.infura.io

thanks

**Comment by mmmmmagina** [comment]
This setting is for configuring Loopring relay url， not the ethnode address.
For now, Only Loopring Foundation runs relay that you saw in the default config.

**Comment by dirmansebastian** [comment]
if i run my geth node, then i start websockets it should work right?

**Comment by mmmmmagina** [comment]
No, it does not work.  Loopring Relay NOT equal eth node. Please take a look at http://loopring.org/documents.html  for a reference.
